python/servo/package_commands.py

```diff
@@ -133,6 +137,25 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 checked.update(checking)
                 need_checked.difference_update(checked)
 
+            print("Adding version to Credits.rtf")
+            version_command = [binary_path, '--version']
+            p = subprocess.Popen(version_command,
+                                 stdout=subprocess.PIPE,
+                                 stderr=subprocess.PIPE,
+                                 universal_newlines=True)
+            version, stderr = p.communicate()
+            if p.returncode != 0:
+                print("Error occurred when getting Servo version: " + stderr)
+                return 1
```

Comment:
I think raising an exception is the right way to signal errors for a Mach `@Command` based on other code, but I couldn't find any definitive docs.